mne/epochs.py

```diff
@@ -809,14 +809,22 @@ def __next__(self, *args, **kwargs):
         """Provide a wrapper for Py3k."""
         return self.next(*args, **kwargs)
 
-    def average(self, picks=None):
-        """Compute average of epochs.
+    def average(self, picks=None, method="mean"):
+        """Compute an aggregate over epochs, by default the mean.
```

Comment:
I would keep the headline `Compute average of epochs.` even if it can be more than this. It's easier to example and what 99.9% people will use.